docker/requirements.in

```diff
@@ -125,6 +126,9 @@ pyTMD==1.0.6
 pyviz_comms==2.0.1
 pyyaml
 rioxarray
+# rsgislib is conda only, but was manually packaged as a binary wheel compatible
+# with geobase docker images
+rsgislib==4.1.95
```

Comment:
This package is manually build and published by DEA [here](https://packages.dea.ga.gov.au/rsgislib/). Installing it require maintenance.  

> rsgislib this was manually done ages ago and only for Python 3.6/3.8, it's a conda only library that was packaged up in a dodgy way a while back and never since updated.

**Question:**
    - Does any DEAfrica notebooks require this library?
    - If not, can we remove this?

Reply:
@fangfy @lgkgh see my comments ⬆️

Reply:
@fangfy @lgkgh see my comments ⬆️

Reply:
Just closed the related issue. No I don't think we need it and it threw an error during import. Easier to just remove.

Reply:
Hi @NikitaGandhi and @fangfy 

Thank you. I will remove rsgislib.